rds/index.js

```diff
@@ -0,0 +1,246 @@
+export function toJsonObject(inputStr) {
+  // on AWS inputStr is always a string, but on LocalStack the input may be an object.
+  let input;
+  try {
+    input = JSON.parse(inputStr);
+  } catch (SyntaxError) {
+    input = inputStr;
+  }
```

Comment:
I'm not 100% happy with this departure from parity here, but I feel this is a compromise to get a solution working on LocalStack. In the future I may revisit this approach and more closely match the AWS behaviour.